### ISSUE TITLE
INVALID_ARGUMENT: Incompatible shapes: [1,96] vs. [1,4020] 	 [[{{node gradients/mul_4_grad/BroadcastGradientArgs}}]] ERROR: DeePMD-kit Error: TensorFlow Error: INVALID_ARGUMENT: Incompatible shapes: [1,96] vs. [1,4020] 	 [[{node gradients/mul_4_grad/BroadcastGradientArgs}]] (/home/conda/feedstock_root/build_artifacts/libdeepmd_1678813995417/work/source/lmp/pair_deepmd.cpp:471)

### ISSUE THREAD

**Issue by gengxingze** [deepmodeling/deepmd-kit#2398]
### Summary

After reinstalling deepkit today, after training the model to generate graph.pb, the following error appeared in lammps, what is the reason? I tried graph.pb that was trained before updating it and it works fine

>>> Info of model(s):
  using   1 model(s): graph.pb 
  rcut in model:      6.5
  ntypes in model:    1

CITE-CITE-CITE-CITE-CITE-CITE-CITE-CITE-CITE-CITE-CITE-CITE-CITE

Your simulation uses code contributions which should be cited:
- USER-DEEPMD package:
The log file lists these citations in BibTeX format.

CITE-CITE-CITE-CITE-CITE-CITE-CITE-CITE-CITE-CITE-CITE-CITE-CITE

Generated 0 of 0 mixed pair_coeff terms from geometric mixing rule
Neighbor list info ...
  update every 1 steps, delay 0 steps, check yes
  max neighbors/atom: 2000, page size: 100000
  master list distance cutoff = 16.5
  ghost atom cutoff = 16.5
  binsize = 8.25, bins = 2 2 2
  1 neighbor lists, perpetual/occasional/extra = 1 0 0
  (1) pair deepmd, perpetual
      attributes: full, newton on
      pair build: full/bin/atomonly
      stencil: full/bin/3d
      bin: standard
Setting up Verlet run ...
  Unit style    : metal
  Current step  : 0
  Time step     : 0.001
INVALID_ARGUMENT: Incompatible shapes: [1,96] vs. [1,4020]
	 [[{{node gradients/mul_4_grad/BroadcastGradientArgs}}]]
ERROR: DeePMD-kit Error: TensorFlow Error: INVALID_ARGUMENT: Incompatible shapes: [1,96] vs. [1,4020]
	 [[{node gradients/mul_4_grad/BroadcastGradientArgs}]] (/home/conda/feedstock_root/build_artifacts/libdeepmd_1678813995417/work/source/lmp/pair_deepmd.cpp:471)
Last command: run  

### DeePMD-kit Version

2.2

### TensorFlow Version

.

### Details

After reinstalling deepkit today, after training the model to generate graph.pb, the following error appeared in lammps, what is the reason? I tried graph.pb that was trained before updating it and it works fine

**Comment by njzjz** [comment]
Fixed in #2390. A new version has been released.
